README.md

```diff
@@ -15,38 +29,51 @@ docker run -it `
 5. `-p 7860:7860` specifies the [published port](https://docs.docker.com/network/).
 6. `--name <container_name>` assigns the container a meaningful name. You can restart the same container (after it exits) by `docker start -i <container_name>`.
-7. `nuullll/ipex-arc-sd:v0.1` specifies the docker image. If it doesn't exist locally, docker will pull from the [corresponding dockerhub registry](https://hub.docker.com/r/nuullll/ipex-arc-sd).
+7. `disty0/sd-webui-ipex:latest` specifies the docker image. If it doesn't exist locally, docker will pull from the [corresponding dockerhub registry](https://hub.docker.com/r/disty0/sd-webui-ipex).
+
+### (For Developers) Build docker image 
 
-### (For Developers) Build docker image in PowerShell
+### In Linux
+
+```sh
+docker build --build-arg UBUNTU_VERSION=22.04 \
+--build-arg PYTHON=python3.10 \
+--build-arg ICD_VER=23.17.26241.21-647~22.04 \
+--build-arg LEVEL_ZERO_GPU_VER=1.3.26241.21-647~22.04 \
+--build-arg LEVEL_ZERO_VER=1.11.0-647~22.04 \
+--build-arg LEVEL_ZERO_DEV_VER=1.11.0-647~22.04 \
+--build-arg DPCPP_VER=2023.1.0-46305 \
+--build-arg MKL_VER=2023.1.0-46342 \
+--build-arg CMPLR_COMMON_VER=2023.1.0 \
+--build-arg DEVICE=flex \
+-t sd-webui-ipex \
+-f Dockerfile .
+```
+
+### In PowerShell on Windows
 
 ```powershell
 docker build --build-arg UBUNTU_VERSION=22.04 `
 --build-arg PYTHON=python3.10 `
 --build-arg ICD_VER=23.17.26241.21-647~22.04 `
---build-arg LEVEL_ZERO_GPU_VER=1.3.25593.18-601~22.04 `
---build-arg LEVEL_ZERO_VER=1.9.4+i589~22.04 `
---build-arg LEVEL_ZERO_DEV_VER=1.9.4+i589~22.04 `
+--build-arg LEVEL_ZERO_GPU_VER=1.3.26241.21-647~22.04 `
+--build-arg LEVEL_ZERO_VER=1.11.0-647~22.04 `
+--build-arg LEVEL_ZERO_DEV_VER=1.11.0-647~22.04 `
 --build-arg DPCPP_VER=2023.1.0-46305 `
 --build-arg MKL_VER=2023.1.0-46342 `
 --build-arg CMPLR_COMMON_VER=2023.1.0 `
---build-arg TORCH_VERSION=1.13.0a0+git6c9b55e `
---build-arg IPEX_VERSION=1.13.120+xpu `
---build-arg TORCHVISION_VERSION=0.14.1a0+5e8e2f1 `
---build-arg IPEX_WHL_URL=https://developer.intel.com/ipex-whl-stable-xpu `
 --build-arg DEVICE=flex `
--t ipex-arc-sd:v0.1 `
+-t sd-webui-ipex `
 -f Dockerfile .
 ```
-
-**Notes: `ICD_VER=23.05.25593.18-601~22.04` is used by the [IPEX xpu Dockerfile](https://github.com/intel/intel-extension-for-pytorch/blob/e413ea5f4501ed9bfc9ff4040b46ff4ce8fca87a/docker/build.sh#L34), which triggers an [OpenCL Out-Of-Memory error](https://github.com/vladmandic/automatic/issues/1474) for me.**
```

Comment:
Please keep this.

Reply:
I updated outdated LEVEL_ZERO versions too.
I don't think this note is necessary but i re-added it in my last commit.